packages/sdk/src/reads/getConvexWrapperEstimateRewards.ts

```diff
@@ -0,0 +1,35 @@
+import { type ReadContractParameters, readContractParameters } from "../utils/viem.js";
+import { IConvexCurveLpStakingWrapperLib } from "@enzymefinance/abis/IConvexCurveLpStakingWrapperLib";
+import { type Address, ContractFunctionExecutionError, type PublicClient } from "viem";
+
+export async function getConvexWrapperEstimateRewards(
```

Comment:
this function should accept an array of `stakingWrappers`. You can create another function that will use this one under the hood